website/source/docs/telemetry/overview.html.md

```diff
@@ -0,0 +1,159 @@
+[StatsD][statsd-telem] and [Circonus][circonus-telem].
+
+## Alerting
+
+The recommended practice for alerting is to leverage the alerting capabilities
+of your monitoring provider. Nomad’s intention is to surface metrics that enable
+users to configure the necessary alerts using their existing monitoring systems
+as a scaffold, rather than to natively support alerting. Here are a few common
+patterns:
+
+* Export metrics from Nomad to Prometheus using the [StatsD
+  exporter][statsd-exporter], define [alerting rules][alerting-rules] in
+  Prometheus, and use [Alertmanager][alertmanager] for summarization and
+  routing/notifications (to PagerDuty, Slack etc..). A similar workflow is
+  supported for [Datadog][datadog-alerting].
+
+* Periodically submit test jobs into Nomad to determine if your application
+  deployment pipeline is working end-to-end. This pattern is well-suited to
+  batch processing workloads.
+
+* Deploy Nagios on Nomad. Centrally manage Nomad job files and add the Nagios
+  monitor when a new Nomad job is added. When a job is removed, remove the
+  Nagios monitor. Map Consul alerts to the Nagios monitor. This provides a
+  job-specific alerting system.
+
+* Write a script that looks at the history of each batch job to determine
+  whether or not the job is in an unhealthy state, updating your monitoring
+  system as appropriate. In many cases, it may be ok if a given batch job fails
+  occasionally, as long as it goes back to passing.
+
+# Key Performance Indicators
+
+The sections below cover a number of important metrics
+
+## Consensus Protocol (Raft)
+
+Nomad uses the Raft consensus protocol for leader election and state
+replication. Spurious leader elections can be caused by networking issues
+between the servers or insufficient CPU resources. Users in cloud environments
+often bump their servers up to the next instance class with improved networking
+and CPU to stabilize leader elections. The nomad.raft.leader.lastContact metric
+is a general indicator of Raft latency which can be used to observe how Raft
+timing is performing and guide the decision to upgrade to more powerful servers.
+nomad.raft.leader.lastContact should not get too close to the leader lease
+timeout of 500ms.
+
+## Federated Deployments (Serf)
+
+Nomad uses the membership and failure detection capabilities of the Serf library
+to maintain a single, global gossip pool for all servers in a federated
+deployment. An uptick in member.flap and/or msg.suspect is a reliable indicator
+that membership is unstable.
+
+## Scheduling
+
+The following metrics allow an operator to observe changes in throughput at the
+various points in the scheduling process (evaluation, scheduling/planning and
+placement):
+
+* **nomad.broker.total_blocked** - The number of blocked evaluations.
+* **nomad.worker.invoke_scheduler.\<type\>** - The time to run the scheduler of
+  the given type. nomad.plan.evaluate - The time to evaluate a scheduler Plan.
```

Comment:
formatting... `nomad.plan.evaluate` should be on a new line

Reply:
Good catch. Thanks.